xt/01-make-update-deps.t

```diff
@@ -1,17 +1,29 @@
 #!/usr/bin/env perl
-# Copyright 2022 SUSE LLC
+# Copyright SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+use strict;
+use warnings 'all';
+
 use Test::Most;
 use Test::Warnings;
 use FindBin '$Bin';
 
+my $hash = {'unnecesary' => 'value'};#intentionally leaving this so tidy & critic fail.
+ok(is(0, 1));#intentionally leaving this so tidy & spec fail.
```

Comment:
if this is just to demonstrate how checks work I suggest to create another draft PR and bring this PR into shape without failing checks

Reply:
Addressed & Squashed all the test commits.